invoker/core/src/main/java/com/google/cloud/functions/invoker/http/HttpResponseImpl.java

```diff
@@ -86,22 +86,31 @@ public OutputStream getOutputStream() throws IOException {
   @Override
   public synchronized BufferedWriter getWriter() throws IOException {
     if (writer == null) {
-      // Unfortunately this means that we get two intermediate objects between the
-      // object we return
-      // and the underlying Writer that response.getWriter() wraps. We could try
-      // accessing the
-      // PrintWriter.out field via reflection, but that sort of access to non-public
-      // fields of
-      // platform classes is now frowned on and may draw warnings or even fail in
-      // subsequent
-      // versions.
-      // We could instead wrap the OutputStream, but that would require us to deduce
-      // the appropriate
-      // Charset, using logic like this:
-      // https://github.com/eclipse/jetty.project/blob/923ec38adf/jetty-server/src/main/java/org/eclipse/jetty/server/Response.java#L731
-      // We may end up doing that if performance is an issue.
+      // Unfortunately this means that we get two intermediate objects between the object we return
+      // and the underlying Writer that response.getWriter() wraps. We could try accessing the
+      // PrintWriter.out field via reflection, but that sort of access to non-public fields of
+      // platform classes is now frowned on and may draw warnings or even fail in subsequent
+      // versions. We could instead wrap the OutputStream, but that would require us to deduce the
+      // appropriate Charset, using logic like this:
+      // https://github.com/eclipse/jetty.project/blob/923ec38adf/jetty-server/src/main/java/org/eclipse/jetty/server/Response.java#L731 We may end up doing that if performance is an issue.
```

Comment:
The comment was weirdly formatted before, and looks much better now, except that this `We may end up...` text should be on a new line.